pyhealth/datasets/configs/mimic3.yaml

```diff
@@ -116,10 +120,18 @@ tables:
   noteevents:
     file_path: "NOTEEVENTS.csv.gz"
     patient_id: "subject_id"
+    join:
+      - file_path: "ADMISSIONS.csv.gz"
```

Comment:
Why do we need to join with ADMISSIONS table here? It seems the `dischtime` wasn't get stored.